microsoft-edge/dev-videos/index.md

```diff
@@ -17,278 +17,818 @@ Microsoft publishes new videos on a regular basis on the [Microsoft Edge YouTube
+* The **More tabs** button.
+* The **More tools** button to see all 30+ tools.
+* Closing tools.
+* Reordering tools in the main toolbar.
+* The **Drawer** and its toolbar.
+* Moving tools between the main toolbar and **Drawer** toolbar.
+* How to restore the default settings.
+
+See also:
+* [Overview of DevTools](../devtools-guide-chromium/overview.md)<!-- Video section is ok -->
+* [About the list of tools](../devtools-guide-chromium/about-tools.md)
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 103
+## Microsoft Edge | What's New in DevTools 104
+
+August 5, 2022
 
-_Published on 5 July 2022._
+[![Thumbnail image for video "What's new in DevTools 104"](./index-images/devtools-whatsnew-104.png)](https://www.youtube.com/watch?v=0_ZprFX8x0I)
 
-[![Thumbnail image for the DevTools What's New in 103 video](./index-images/devtools-whatsnew-103.png)](https://www.youtube.com/watch?v=aa0C6BRdaPA)
+Covers:
+* Rendering/accessibility bug fixes.
+* Composited layers are now in the 3D View tool and the Layers tool was removed.
+* Group files in the Sources tool.
+* Find where nodes get slotted in web components.
+* Where to find more information.
+* Configure the Activity Bar location in Focus Mode.
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 103](../devtools-guide-chromium/whats-new/2022/06/devtools-103.md).
+See also:
+* [What's New in DevTools 104](../devtools-guide-chromium/whats-new/2022/08/devtools-104.md)<!-- Video section is ok -->
+* [Debug the web in 3D with the 3D View tool](https://aka.ms/debug-in-3d) - blog post.
 
 
 <!-- ====================================================================== -->
-## DevTools - Debug the web in 3D
+## Microsoft Edge | What's New in DevTools 103
+
+July 5, 2022
+
+[![Thumbnail image for video "What's new in DevTools 103"](./index-images/devtools-whatsnew-103.png)](https://www.youtube.com/watch?v=aa0C6BRdaPA)
 
-_Published on 21 June 2022._
+Covers:
+* Redesigned **Welcome** tool.
+* New color picker.
+* New memory heap snapshot node types.
+* High contrast mode bug fix.
+* Open any HTML page in the VS Code extension.
 
-[![Thumbnail image for the DevTools 3D View tool video](./index-images/3d-view-tool.png)](https://www.youtube.com/watch?v=LVb1Xzzd72w)
+See also:
+* [What's New in DevTools 103](../devtools-guide-chromium/whats-new/2022/06/devtools-103.md)<!-- Video section is ok -->
+
+
+<!-- ====================================================================== -->
+## Debug the web in 3D
+
+June 21, 2022
+
+[![Thumbnail image for video "Debug the web in 3D"](./index-images/3d-view-tool.png)](https://www.youtube.com/watch?v=LVb1Xzzd72w)
 
 The **3D View** tool in Microsoft Edge DevTools provides a 3-dimensional representation of the webpage you're inspecting.  Use the **3D View** tool to debug common web development problems such as:
 
+Covers:
```

Comment:
It feels a bit odd here to have two leading sentences with colons.

* "Use the 3D View tool to debug common web development problems such as:"
* "Covers:"

Should we remove "Covers:" for a nicer transition?

Reply:
```
The **3D View** tool in Microsoft Edge DevTools provides a 3-dimensional representation of the webpage you're inspecting.  Use 
the **3D View** tool to debug common web development problems, 
such as:
* Deeply nested DOM nodes.
* Out-of-document elements.
* Unwanted scrollbars.
* Z-index stacking issues.
* Composited layers performance.
```